src/app/screens/Onboard/ConnectGaloy/index.tsx

```diff
@@ -0,0 +1,241 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import axios from "axios";
+import Input from "../../../components/Form/Input";
+import Button from "../../../components/Button";
+
+import utils from "../../../../common/lib/utils";
+
+const url = process.env.GALOY_URL || "https://api.staging.galoy.io/graphql/";
+const defaultHeaders = {
+  Accept: "application/json",
+  "Access-Control-Allow-Origin": "*",
+  "Content-Type": "application/json",
+};
+
+export default function ConnectGaloy() {
+  const navigate = useNavigate();
+  const [loading, setLoading] = useState(false);
+  const [phoneNumber, setPhoneNumber] = useState<string>("");
```

Comment:
Typescript will infer the type as string while passing "", so no need to add "<string>"

Reply:
Addressed in #714